public/assets/temporary/custom-style.css

```diff
@@ -68,4 +68,4 @@ h5 {
     align-self: stretch;
     height: 0;
     border: 1px var(--color-neutrals-blue) solid;
-}
\ No newline at end of file
+}
```

Comment:
i could revert this but my ide / prettier will always "correct" this file upon saving. So this will appear anyways when somebody changes this file again